mandatory/4-tax.js

```diff
@@ -5,7 +5,12 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price * 0.2;
```

Comment:
I wrote the same way as you, but I did not pass the test. So I had to calculated total price including the tax in this function calculateSalesTax(). I wrote: return 1.2 * price; and the test then passed.